backend/src/main/java/co/inventorsoft/scripty/exception/ScriptyException.java

```diff
@@ -0,0 +1,25 @@
+package co.inventorsoft.scripty.exception;
+
+/**
+ * ScriptyException 
+ * All project's custom exceptions should use or extend it
+ */
+public class ScriptyException extends Exception {
```

Comment:
In most cases when you deal with checked exceptions you want to get rid of them :). It also would be better to handle exceptions globally via Spring's exception handling infrastructure https://spring.io/blog/2013/11/01/exception-handling-in-spring-mvc. Therefore I think we don't need a checked wrapper for exceptions